docker-compose.yml

```diff
@@ -98,8 +98,12 @@ services:
     image: dojot/auth
     restart: always
     depends_on:
-      - mongodb
       - apigw
+      - postgres
+    environment:
+      DB_HOST: "postgres"
+      DB_USER: "kong"
```

Comment:
I know... this not good. I can´t find a way to create multiples database users on the Postgres container.